packages/react-events/src/dom/__tests__/Press-test.internal.js

```diff
@@ -698,10 +612,8 @@ describe.each(environmentTable)('Press responder', hasPointerEvents => {
         'onPressStart',
         'onPressChange',
         'onPressMove',
-        'onPressEnd',
-        'onPressChange',
-        'onPressStart',
-        'onPressChange',
+        'onPressMove',
```

Comment:
This relates to my comment above, surely if you move out, then end should fire?

Reply:
We can do that for events that aren't capturing the target. For now this is fine and consistent across pointers